### DISCUSSION TITLE
part-ii/2012-63/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#188] [Question comments]
Part II 2012 Galois Theory: Paper 1, Section II, 18H 

https://questions.tripos.org/part-ii/2012-63/

**Comment by eha1234** [comment]
<details><summary>Spoilers</summary>

Note $(\mathbb{Z}/20\mathbb{Z})^*$ isn't cyclic. In fact, it's isomorphic to $C_4 \times C_2$ 
(I think). I get 4 proper subfields. 
</details>
